frontend/src/components/Contexts/SafeZoneContext.tsx

```diff
@@ -3,6 +3,8 @@ import { createContext, FC, useContext, useState } from 'react'
 interface ISafeZoneContext {
     safeZoneStatus: boolean
     switchSafeZoneStatus: (newSafeZoneStatus: boolean) => void
+    safeZoneBanner: boolean
```

Comment:
Could we maybe use some other name which explains why it is a boolean? safeZoneBannerActive or something like that for example?